oscal-publish/setup/GRAB-OSCAL-CATALOG-SCHEMA.xpl

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc"
+   xmlns:c="http://www.w3.org/ns/xproc-step" version="3.0"
+   xmlns:ox="http://csrc.nist.gov/ns/oscal-xproc3"
+   xmlns:xs="http://www.w3.org/2001/XMLSchema"
+   type="ox:GRAB-OSCAL-CATALOG-SCHEMA"
+   name="GRAB-OSCAL-CATALOG-SCHEMA">
+
+<!--
+
+Since the OSCAL Schema files are directly addressable as resources over the Internet
+(thank you AJS, NW, CC) we simple pull them straight in, no load ...-->
+  
+   <p:variable name="download-path" select="'https://github.com/usnistgov/OSCAL/releases/download/v1.1.2'"/>
+   
+   <p:variable name="destination" select="resolve-uri('../lib', static-base-uri())"/>
```

Comment:
Unless you want to show the base explicitly for emphasis, you can use the one-argument form of `resolve-uri` here and in other locations of this repo. The base defaults to the static base URI (https://www.w3.org/TR/xpath-functions-31/#func-resolve-uri).